frontend/www/js/omegaup/components/course/ViewProgress.vue

```diff
@@ -192,7 +193,8 @@ export function toOds(courseName: string, table: TableCell[][]): string {
 })
 export default class CourseViewProgress extends Vue {
   @Prop() course!: types.CourseDetails;
-  @Prop() completeStudentsProgress!: types.StudentProgressInCourse[];
+  @Prop({ default: null })
+  completeStudentsProgress!: types.StudentProgressInCourse[];
```

Comment:
```suggestion
  completeStudentsProgress!: types.StudentProgressInCourse[] | null;
```

si esto legalmente puede tener `null` como valor (según el default), hay que declararlo acordemente.